docs/user/integration/prometheus/README.md

```diff
@@ -0,0 +1,154 @@
+# Integrate with Prometheus
+
+| Category     |                                         |
+| ------------ | --------------------------------------- |
+| Signal types | metrics                                 |
+| Backend type | custom local                            |
+| OTLP-native  | yes                                     |
+
+Learn how to configure the Telemetry module to ingest metrics in a custom [Prometheus](https://prometheus.io/) instance deployed with the [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack).
+
+## Table of Content
+
+- [Integrate with Prometheus](#integrate-with-prometheus)
+  - [Table of Content](#table-of-content)
+  - [Prerequisites](#prerequisites)
+  - [Context](#context)
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- The [Telemetry module](../../README.md) is added. For details, see [Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install). <!-- This link differs for OS and SKR -->
```

Comment:
Do you want me to add this doc to Help Portal?

Reply:
The prometheus integration guide? no, let's focus on SAP scenarios only there